app/views/devise/sessions/new.html.erb

```diff
@@ -6,13 +6,19 @@
 
     <p><%= t('devise.sessions.new.password_signin') %></p>
 
+    <!-- 
```

Comment:
Let's make this a ERB ruby comment instead of an html comment -- `<%# whatever %>

Reply:
it wasn't seeming to fool the i18n tasks script as ERB ruby, but I'll give it another go!

Reply:
oh word -- in that case, stick these in the Devise controller or Oauth controller as comments, per https://github.com/glebm/i18n-tasks#fine-tuning